### PR TITLE
Schema Changes

### PR DESCRIPTION
**Pull request by joedevivo**

### CODE REVIEW

**Comment by joedevivo** [comment]
@seancribbs @angrycub please sign off at your convenience

File: priv/eleveldb.schema [hunk 1]
```diff
@@ -51,303 +70,199 @@
...
+{mapping, "multi_backend.$name.leveldb.write_buffer_size_max", "riak_kv.multi_backend", [
+  {default, "30MB"},
+  {datatype, bytesize},
+  {level, advanced}
+]}.
 
 %% @doc Each database .sst table file can include an optional "bloom
 %% filter" that is highly effective in shortcutting data queries that
-%% are destined to not find the requested key. The bloom_filter
+%% are destined to not find the requested key. The bloom filter
```
Comment (angrycub) [inline priv/eleveldb.schema]: Bloom should be capitalized, since it's a proper name.  Just a nitpick

**Comment by angrycub** [comment]
Other than pedantic nitpick... this is a +1 from me.

**Comment by joedevivo** [comment]
@angrycub fixed, merging.

**Pull request merged**